docs/development/customizing-bootstrap.mdx

```diff
@@ -24,6 +25,6 @@ For example to use the border-radius mixin:
 
 Make sure you import only partials and not main Sass files, otherwise you will end up generating duplicate CSS which might cause issues.
 
-To change the default Bootstrap settings like colors, border-radius, etc, add or change the value of the property in the partial file `src/main/webapp/content/scss/_bootstrap-variable.scss`
+To change the default Bootstrap settings like colors, border-radius, etc., add or change the value of the property in the partial file `src/main/webapp/content/scss/_bootstrap-variables.scss`
 
 All values defined in Bootstrap [_variables.scss](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss) can be overwritten here.
```

Comment:
This should link to the v5 version of `_variables.scss`.

Reply:
ah, I missed that - fixed